bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerChecker.java

```diff
@@ -51,6 +51,14 @@
     public final BookieClient bookieClient;
     public final BookieWatcher bookieWatcher;
 
+    private static int averageEntrySize;
+
+    private static final int INITIAL_AVERAGE_ENTRY_SIZE = 1024;
```

Comment:
consider simplifying by using number of entries in flight (using Semaphore, releasing when processed) instead of guessing avg sizes and rate limiting. Or rate limit by number of entries.
This also removes need in synchronization.

One ledger may have entries of 512K, next one of the 1K, third one is mixed. 
I don't see how tracking avg size significantly helps in this case, especially if the backpressure is not enabled.

Reply:
OK  I will fix

Reply:
I will rate limit by number of entries

Reply:
In accordance with your suggestions, I made changes。
Please review again, thank you！@dlg99